apps/super-app/src/routes/SupportDetailWip.tsx

```diff
@@ -17,10 +18,25 @@ import './SupportDetailWip.css';
 
 function Header(props: { title: string }) {
   const { title } = props;
+  const navigate = useNavigate();
   return (
     <nav className="support-header">
-      <ArrowBackIosNewIcon />
-      <Typography variant="h4">{title}</Typography>
+      <IconButton
+        aria-label="back"
+        sx={{ color: 'text.primary' }}
+        onClick={() => navigate(-1)}
+      >
+        <ArrowBackIosNewIcon />
+      </IconButton>
+      <Typography
+        component="div"
```

Comment:
Why component="div"? `div` is semantically meaningless, whereas leaving it unspecified will at least have _some_ meaning.